src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -566,8 +566,8 @@ const MetaTraderUI = (() => {
     };
 
     const showHideFinancialAuthenticate = (acc_type) => {
-        if (MetaTraderConfig.hasAccount(acc_type) && accounts_info[acc_type].account_type === 'financial') {
-            $('#financial_authenticate_msg').setVisibility(!MetaTraderConfig.isAuthenticated());
+        if (MetaTraderConfig.hasAccount(acc_type)) {
+            $('#financial_authenticate_msg').setVisibility(MetaTraderConfig.isAuthenticationPromptNeeded());
```

Comment:
same here, can we check that mt account type is advanced and revert to is authenticated check?